common/build/build-common/api-extractor-base.json

```diff
@@ -18,9 +18,10 @@
 	 */
 	"apiReport": {
 		"enabled": true,
-		"reportFileName": "<unscopedPackageName>.api.md",
+		"reportFileName": "<unscopedPackageName>",
```

Comment:
The `.api.md` extension postfix is no longer required, and support for it will be removed in the future.